app/models/course/material/folder/download.rb

```diff
@@ -0,0 +1,7 @@
+class Course::Material::Folder::Download < ActiveRecord::Base
```

Comment:
Thanks @evelynleems !

@lowjoel Since the zip files are cached and they will expire, I think I don't need to add anything additional here ?